packages/x-components/src/components/__tests__/base-grid.spec.ts

```diff
@@ -67,7 +67,7 @@ function renderBaseGridComponent({
 describe('testing Base Grid', () => {
   it('allows configuring the number of columns and updates the css class accordingly', () => {
     const { wrapper } = renderBaseGridComponent({ columns: 5 });
-    expect(wrapper.classes()).toContain(`x-base-grid--cols-5`);
+    expect(wrapper.classes()).toContain(`x-grid--cols-5`);
```

Comment:
Should we keep the old `base` classes too? Or are we already working to remove them?

Reply:
we remove it here, in this component, in this task, ONLY, at the moment